BaseSpace.SDK/ServiceModels/Request/ListProjectsRequest.cs

```diff
@@ -7,6 +7,8 @@ public class ListProjectsRequest : AbstractResourceListRequest<ListProjectsRespo
     {
         public string Name { get; set; }
 
+        public string[] Include { get; set; }        
```

Comment:
this makes the include constants hard to discover currently. Perhaps add a static class "ProjectIncludes" with the one string constant we have currently "Permissions"?